### PR TITLE
Poster project second pass

### PR DESCRIPTION
**Pull request by djdv**
Reduced the list of topics down to 30.
TODO:
- Selected topics need to be broken out into one page documents.
- Remove the alternate section (or leave it?)
><TODO: Insert video reel from how it went in the Dev Meetings>

I'm not sure if we have any video from this session. It may have to be populated from this year's event. 👀

### CODE REVIEW

**Review by daviddias — APPROVED** [review]
Looking good!

Will you add a column with Owner/Champion next?

File: POSTER_PROJECTS/README.md [hunk 1]
```diff
@@ -53,53 +53,38 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
...
+| 9 | [js-ipfs Infrastructure](topic-js-ipfs-infra.md) | TBD |
+| 10 | [Mutable File System (MFS)](topic-MFS.md) | TBD |
+| 11 | [KeyStore](topic-keystore.md) | TBD |
+| 12 | [IPFS Nodes](topic-IPFS-node.md) | TBD |
+| 13 | [Rendezvous Protocol](topic-rendezvous-protocol.md) | TBD |
+| 14 | [Libp2p Connection Flow](topic-libp2p.md) | TBD |
+| 15 | [Circuit Relay](topic-circuit-relay.md)| TBD |
+| 16 | [IPFS DHT](topic-IPFS-DHT.md) | TBD |
+| 17 | [IPNS](topic-IPNS.md) | TBD |
+| 18 | [Content Provider Records]( | how do nodes provide content? What options exist?|
```
Comment (daviddias) [inline POSTER_PROJECTS/README.md]: missing `)`

File: POSTER_PROJECTS/README.md [hunk 2]
```diff
@@ -53,53 +53,38 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
 - Take a flip chart, markers, etc and work wherever you want to work.
 
 ## Topics
-
-| # | Topic | Description |
+| # | Topic | Owner |
```
Comment (daviddias) [inline POSTER_PROJECTS/README.md:57]: @djdv Can you add the owners to each topic soon? Chris needs that info for the website

Comment (djdv) [reply POSTER_PROJECTS/README.md:57]: Status will be posted here: https://gist.github.com/djdv/f8712c70211848e7af23cd8d86c2b320
and committed to the branch when confirmed.

File: POSTER_PROJECTS/topic-dweb-addressing.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- What is the current state of dweb protocol and address handling in web browsers?
+- What is needed for other vendors to implement support?
+- How have IPFS nodes handled migrating to browser compatible addresses?
+- What challenges have there been in handling compatibility?
+
+## Where to learn about it
+
+<https://github.com/ipfs/ipfs/issues/337>  
+<https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md>  
+<https://bugzilla.mozilla.org/show_bug.cgi?id=1428446>
```
Comment (lidel) [inline POSTER_PROJECTS/topic-dweb-addressing.md:22]: Adding some additional references for "DWeb Addressing" poster sesssion:
```suggestion
<https://bugzilla.mozilla.org/show_bug.cgi?id=1428446>  
<https://github.com/ipfs/in-web-browsers/issues/147>
<https://bugs.chromium.org/p/chromium/issues/detail?id=64100>
<https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/29sFh4tTdcs>
<https://github.com/whatwg/html/pull/3936>
```

File: POSTER_PROJECTS/README.md [hunk 4]
```diff
@@ -52,54 +52,40 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
...
-| 41 | go-ipfs Infrastructure | [go-ipfs-infra](topic-go-ipfs-infra.md) |
-| 42 | IPFS Nodes | What makes an IPFS node an IPFS node? List specs that need to be implemented, examples of (non-PL) implementations (whole or individual components). |
-| 43 | History? | Outline some of the historical changes in IPFS, such as the extraction of libp2p, changes in standards, etc. |
-| 44 | js-ipfs Infrastructure | [js-ipfs-infra](topic-js-ipfs-infra.md)
+| 1 | [UnixfsV1](topic-unixfsv1.md) | TBD |
+| 2 | [UnixfsV2](topic-unixfsv2.md) | @mikeal |
+| 3 | [The design of the IPFS Repo](topic-IPFS-repo.md) | TBD |
+| 4 | [Swarming (concept)](topic-swarming.md) | TBD |
+| 5 | [Bitswap](topic-bitswap.md) | @jacobheun |
+| 6 | [Pinning & GC on IPFS](topic-pinning-gc.md)| TBD |
```
Comment (daviddias) [inline POSTER_PROJECTS/README.md]: Ping @Kubuxu 

```suggestion
| 6 | [Pinning & GC on IPFS](topic-pinning-gc.md)| @Kubuxu |
```

File: POSTER_PROJECTS/README.md [hunk 5]
```diff
@@ -52,54 +52,40 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
...
+| 10 | [Mutable File System (MFS)](topic-MFS.md) | @leshokunin |
+| 11 | [KeyStore](topic-keystore.md) | TBD |
+| 12 | [IPFS Nodes](topic-IPFS-node.md) | @carsonfarmer |
+| 13 | [Rendezvous Protocol](topic-rendezvous-protocol.md) | TBD |
+| 14 | [Libp2p Connection Flow](topic-libp2p.md) | @raulk |
+| 15 | [Circuit Relay](topic-circuit-relay.md)| @ianopolous |
+| 16 | [IPFS DHT](topic-IPFS-DHT.md) | @gpestana |
+| 17 | [IPNS](topic-IPNS.md) | @aschmahmann |
+| 18 | [IPFS/Libp2p content routing](topic-content-provider-records.md) | @bigs |
+| 19 | [IPFS/libp2p PubSub](topic-pubsub.md) | TBD |
```
Comment (daviddias) [inline POSTER_PROJECTS/README.md]:
```suggestion
| 19 | [IPFS/libp2p PubSub](topic-pubsub.md) | @whyrusleeping  |
```

File: POSTER_PROJECTS/README.md [hunk 6]
```diff
@@ -52,54 +52,40 @@ Poster teams should have a maximum of 4 people and a minimum of 2. People will c
...
+| 20 | [IPFS Cluster](topic-cluster.md) | @meiqimichelle (& maybe @hsanjuan) |
+| 21 | [IPFS HTTP API libs](topic-HTTP-API.md)  | @hugomrdias |
+| 22 | [Core API & ipfsx](topic-highlevel-apis.md) | TBD |
+| 23 | [CRDTs](topic-CRDT.md) | TBD |
```
Comment (daviddias) [inline POSTER_PROJECTS/README.md]:
```suggestion
| 23 | [CRDTs](topic-CRDT.md) | @parkan  |
```

**Pull request merged**